### PR TITLE
refactor code structure of graphlearn

### PR DESCRIPTION
**Pull request by goldenleaves**

### CODE REVIEW

File: graphlearn/core/operator/sampler/test/alias_method_unittest.cpp [hunk 1]
```diff
@@ -1,63 +0,0 @@
-/* Copyright 2020 Alibaba Group Holding Limited. All Rights Reserved.
```
Comment (baoleai) [inline graphlearn/core/operator/sampler/test/alias_method_unittest.cpp:1]: why delete this ut?

Comment (goldenleaves) [reply graphlearn/core/operator/sampler/test/alias_method_unittest.cpp:1]: restored

**Comment by baoleai** [comment]
LGTM.

**Review by baoleai — APPROVED** [review]
lgtm.

**Comment by baoleai** [comment]
cc @sighingnow ,

**Review by Seventeen17 — APPROVED** [review]

**Pull request merged**